src/gui/shell.cpp

```diff
@@ -835,6 +838,18 @@ void Shell::handleNeovimNotification(const QByteArray &name, const QVariantList&
 			} else {
 				emit neovimFullScreen(variant_not_zero(args.at(1)));
 			}
+		} else if (guiEvName == "WindowFrameless" && args.size() == 2) {
+			if (isWindow()) {
+				auto isOn = variant_not_zero(args.at(1));
```

Comment:
nitpick: Use the real type + a descriptive name.

```suggestion
				const bool isWindowFrameOn{ variant_not_zero(args.at(1)) };
```